### PR TITLE
Test coverage extension

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ffb8d42653bce4c65b3235ae3ed090b5e0bacd6a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 53ea62143d9e3297b357b28907b6989e5f7d38ca:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 63ad93b2fccdcc854c5b1f3821212cf639def47c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0fc8c7131cdf91d1279d25686482f0b06dd6ed45:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cb01c00935a2c2438341e70751204787ddea9dfb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav This is ok to merge. I have just added a blocker for 5.5 for the dropdown test.

**Comment by dajoRH** [comment]
Lint report for commit 04836622edc395adfb61313efdcfc7d03695232d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/cloud/test_providers.py [hunk 1]
```diff
@@ -259,3 +259,13 @@ def test_api_port_max_character_validation(request):
 
     request.addfinalizer(prov.delete_if_exists)
     prov.create()
+
+
+@pytest.mark.ignore_stream("5.2", "5.3", "5.4")
```
Comment (psav) [inline cfme/tests/cloud/test_providers.py]: I have seen this cause a problem before, wouldn't an uncollectif  > 5.5 be better here?

Comment (mfalesni) [inline cfme/tests/cloud/test_providers.py]: This looks shorter to me. Dunno :)

**Comment by dajoRH** [comment]
Lint report for commit cd189abcc936ff90095e51a25d6b220b4814ea55:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**